<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasuringSystemAddFormClass</class>
 <widget class="QWidget" name="MeasuringSystemAddFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add measuring system</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/icons8-микроскоп-48.png</normaloff>icons/icons8-микроскоп-48.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Data Input</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>120</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>333</width>
            <height>0</height>
           </size>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>105</x>
             <y>4</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>65</y>
             <width>141</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Main contributor name:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>75</x>
             <y>35</y>
             <width>70</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="NameInput">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>10</y>
             <width>161</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="DescriptionInput">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>40</y>
             <width>161</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="MainContributorNameInput">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>70</y>
             <width>161</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tableSelector">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="allElementsTable">
       <property name="minimumSize">
        <size>
         <width>771</width>
         <height>192</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="downButton">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>▼</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="upButton">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>▲</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="chosenElementsTable">
       <property name="minimumSize">
        <size>
         <width>771</width>
         <height>151</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>51</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="AddMeasuringSystemButton">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>Add measuring system</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
